etc/config/c++.amazon.properties

```diff
@@ -1,4 +1,4 @@
-compilers=&gcc86:&icc:&icx:&clang:&clangx86trunk:&rvclang:&wasmclang:&cl:&cross:&ellcc:&zapcc:&djggp:www.godbolt.ms@443:&armclang32:&armclang64:&zigcxx:&cxx6502
+compilers=&gcc86:&icc:&icx:&clang:&clangx86trunk:&llvm-amdgpu:&rvclang:&wasmclang:&cl:&cross:&ellcc:&zapcc:&djggp:www.godbolt.ms@443:&armclang32:&armclang64:&zigcxx:&cxx6502
```

Comment:
In other places we call this by the compiler (llvm is the library we build and link as a client library elsewhere). I already made this rename in other places, can you do it here please?

Reply:
Sure.